build.gradle

```diff
@@ -106,6 +102,7 @@ subprojects {
   gradle.projectsEvaluated {
     tasks.withType(JavaCompile) {
       options.compilerArgs += [
+        '-encoding', 'UTF-8',
```

Comment:
Can be removed

Reply:
Java code shouldn't have any non-ASCII chars anyway (Scala looking at your `λ`s)

Reply:
Done, removed on a follow-up PR